creator-node/src/services/stateMachineManager/stateReconciliation/updateReplicaSet.jobProcessor.ts

```diff
@@ -628,11 +629,9 @@ const _issueUpdateReplicaSetOp = async (
         })
       }
 
-      const {
-        [primary]: oldPrimarySpId,
-        [secondary1]: oldSecondary1SpId,
-        [secondary2]: oldSecondary2SpId
-      } = ContentNodeInfoManager.getCNodeEndpointToSpIdMap()
+      const oldPrimarySpId = cNodeEndpointToSpIdMap.get(primary)
+      const oldSecondary1SpId = cNodeEndpointToSpIdMap.get(secondary1)
+      const oldSecondary2SpId = cNodeEndpointToSpIdMap.get(secondary2)
```

Comment:
nit any reason to do it this way vs on the left?

Reply:
since it's an Map rather than an object we can't destructure anymore